packages/editor/src/lib/utils/sync/TLLocalSyncClient.test.ts

```diff
@@ -69,43 +69,7 @@ test('the client connects on instantiation, announcing its schema', async () =>
 	expect(channel.postMessage).toHaveBeenCalledTimes(1)
 	const [msg] = channel.postMessage.mock.calls[0]
 
-	expect(msg).toMatchObject({ type: 'announce', schema: { recordVersions: {} } })
-})
-
-test('when a client receives an announce with a newer schema version it reloads itself', async () => {
```

Comment:
afaict this behaviour is still there - there's nothing removing it in `TLLocalSyncClient` at least. Why are we removing these?

Reply:
i brought these back